src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker_Patterns.cs

```diff
@@ -17,19 +17,6 @@ namespace Microsoft.CodeAnalysis.CSharp
 {
     internal sealed partial class NullableWalker
     {
-        /// <summary>
-        /// Learn something about the input from a test of a given expression against a given pattern.  The given
-        /// state is updated to note that any slots that are tested against `null` may be null.
-        /// </summary>
-        /// <returns>true if there is a top-level explicit null check</returns>
-        private void LearnFromAnyNullPatterns(
```

Comment:
:memo: This was meant to learn from pure null tests, but `LearnFromDecisionDag` already handles those (and more). This comment applies to other removals in this file.